net/message/message.go

```diff
@@ -106,14 +107,14 @@ func (m *QueryResponse) Type() Type {
 
 // SendKeysMessage is sent by both parties to each other to initiate the protocol
 type SendKeysMessage struct {
-	OfferID            types.Hash
-	ProvidedAmount     float64
-	PublicSpendKey     string
-	PublicViewKey      string
-	PrivateViewKey     string
-	DLEqProof          string
-	Secp256k1PublicKey string
-	EthAddress         string
+	OfferID            types.Hash   `json:"offerID"`
```

Comment:
why are JSON tags added to only this message? seems unnecessary

Reply:
I thought our goal was to use consistent naming in all of our JSON handling. I noticed that this struct didn't have tags and I didn't want to write "ProvidedAmount" (with a capital "P") in the code that checks and generates an error message if this field is invalid.

Reply:
I'll go through all the message types in a future PR, make sure they have consistent JSON tags, and, more importantly, make sure we are doing all needed input validations on all the fields.